core/storage/story/gae_models_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Oppia story models."""
+import datetime
+
+from core.platform import models
+from core.tests import test_utils
+
+(story_models, base_models, ) = models.Registry.import_models(
+    [models.NAMES.story, models.NAMES.base_model])
+
+
+class StoryModelTest(test_utils.GenericTestBase):
+    """Tests for Oppia story models."""
+    def test_story_model(self):
+        """Method to test the story model."""
+
+        committer_id = 'test_committer_id'
+        commit_message = 'test_commit_message'
+        commit_cmds = [{'cmd': 'test_command'}]
+
+        story_instance = story_models.StoryModel(
+            id='id',
+            title='title',
+            description='description',
+            notes='notes',
+            language_code='language_code')
+
+        story_instance.commit(committer_id, commit_message, commit_cmds)
+        story_by_id = story_models.StoryModel.get_by_id('id')
+        self.assertEqual(story_by_id.description, 'description')
+        self.assertEqual(story_by_id.id, 'id')
+        self.assertEqual(story_by_id.notes, 'notes')
+        self.assertEqual(story_by_id.language_code, 'language_code')
+        self.assertEqual(story_by_id.title, 'title')
+
+
+class StorySummaryModelTest(test_utils.GenericTestBase):
+    """Tests for Oppia story summary models."""
+
+    def test_story_summary_model(self):
+        """Method to test the story_summary_model."""
```

Comment:
I think story_summary_model should be changed to StorySummaryModel (That's how the model class is named right?).

Reply:
@nithusha21 Done>